pagopa/io-app
# feat(Redesign Servizi): [IARS-24] Update profile services preference mode

## Short description
This PR adds few points about updating profile services preference mode

### ServicesHome - show and update the current preference (network delay 1 sec)
https://user-images.githubusercontent.com/822471/124297629-8c764480-db5b-11eb-9678-c30bcb67fa36.mov

### ServicesHome - show and update the current preference (network delay 1 sec + error on updating)
https://user-images.githubusercontent.com/822471/124297665-95ffac80-db5b-11eb-9ba6-94069f621d55.mov

### Profile Preferences - current preference on item + updating screen (network delay 1 sec)
https://user-images.githubusercontent.com/822471/124278593-bc671d00-db46-11eb-8b68-0c6f0e77e9fe.mov

### Profile Preferences - current preference on item + updating screen (network delay 1 sec + error on updating)
https://user-images.githubusercontent.com/822471/124278658-cf79ed00-db46-11eb-86a2-9114508127a5.mov

#### how to test
- make sure to have the last commit of dev-server (master)
- make sure to have the feature flag `servicesRedesignEnabled` enabled
